frontend/public/index.html

```diff
@@ -5,7 +5,18 @@
     <base href="[[ .BasePath ]]">
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
-    <title>Tectonic Console</title>
+    [[ if eq .LogoImageName "origin" ]]
```

Comment:
I'm considering renaming `--logo-image-name` this to something more general like `--edition`

Reply:
IS 'edition' a term that is becoming popular?

Reply:
i don't know if that's the best term, maybe `branding`?

Reply:
`branding` is pretty clear, I tend toward that preference.